locations/spiders/key_food_us.py

```diff
@@ -0,0 +1,89 @@
+            item["ref"] = location["name"]
+            item["name"] = location["displayName"]
+            item["street_address"] = ", ".join(filter(None, [location["line1"], location["line2"]]))
+            item["website"] = location["siteUrl"] + location["url"].split("?", 1)[0]
+
+            if "www.keyfood.com" in item["website"] or "keyfoodstores.keyfood.com" in item["website"]:
+                item["brand"] = "Key Food"
+                item["brand_wikidata"] = "Q6398037"
+            elif "keyfoodmarketplace.keyfood.com" in item["website"]:
+                item["brand"] = "Key Food Marketplace"
+                # item["brand_wikidata"] = ""
+            elif "urbanmarketplace.keyfood.com" in item["website"]:
+                item["brand"] = "Key Food Urban Marketplace"
+                # item["brand_wikidata"] = ""
+            elif "superfresh.keyfood.com" in item["website"]:
+                item["brand"] = "SuperFresh"
+                # item["brand_wikidata"] = ""
+            elif "galafoods.keyfood.com" in item["website"]:
+                item["brand"] = "Gala Foods"
+                # item["brand_wikidata"] = ""
+            elif "galafresh.keyfood.com" in item["website"]:
+                item["brand"] = "GalaFresh Farms"
+                # item["brand_wikidata"] = ""
+            elif "fooduniverse.keyfood.com" in item["website"]:
+                item["brand"] = "Food Universe"
+                # item["brand_wikidata"] = ""
+            elif "foodemporium.keyfood.com" in item["website"]:
+                item["brand"] = "The Food Emporium"
+                # item["brand_wikidata"] = ""
+            elif "fooddynasty.keyfood.com" in item["website"]:
+                item["brand"] = "Food Dynasty"
+                # item["brand_wikidata"] = ""
+            elif "tropicalsupermarket.keyfood.com" in item["website"]:
+                item["brand"] = "Tropical Supermarket"
+                # item["brand_wikidata"] = ""
+            elif "halseytradersmarket.keyfood.com" in item["website"]:
+                item["brand"] = "Halsey Traders Market"
+                # item["brand_wikidata"] = ""
+            elif "marketplace.keyfood.com" in item["website"]:
+                item["brand"] = "Marketplace"
+                # item["brand_wikidata"] = ""
+            elif "ernestklein.keyfood.com" in item["website"]:
+                item["brand"] = "Ernest Klein"
+                # item["brand_wikidata"] = ""
+
+            apply_category(Categories.SHOP_SUPERMARKET, item)
+
+            item["opening_hours"] = OpeningHours()
+            if location.get("open24Hours"):
+                item["opening_hours"].add_days_range(DAYS, "00:00", "23:59")
+            else:
+                for day_name, day_hours in location["openings"].items():
+                    item["opening_hours"].add_range(
+                        day_name, day_hours.split(" - ", 1)[0], day_hours.split(" - ", 1)[1], "%I:%M %p"
```

Comment:
I just learnt you can do this:
```suggestion
                        day_name, *day_hours.split(" - ", 1), "%I:%M %p"
```

Reply:
This way you only have to call `split` ones. While it's not exactly a long operation it is a bit pointless.

Reply:
Thanks, I like that approach. The pattern is repeated very frequently so it'll be a good way to simplify spider code.